File: src/main/java/toy/bookchat/bookchat/domain/book/service/dto/response/BookResponse.java
package toy.bookchat.bookchat.domain.book.service.dto.response;

import java.util.List;
import lombok.Builder;
import lombok.Getter;

@Getter
public class BookResponse {

    private String isbn;
    private String title;
    private String publishAt;
    private List<String> authors;
    private String publisher;
    private String bookCoverImageUrl;

    @Builder
    private BookResponse(String isbn, String title, String publishAt, List<String> authors,
        String publisher,
        String bookCoverImageUrl) {
        this.isbn = isbn;
        this.title = title;
        this.publishAt = publishAt;
        this.authors = authors;
        this.publisher = publisher;
        this.bookCoverImageUrl = bookCoverImageUrl;
    }
}

File: android/BookChat/app/src/main/java/com/example/bookchat/data/BookResponse.kt
package com.example.bookchat.data

import com.google.gson.annotations.SerializedName

//DTO
data class BookResponse(
    @SerializedName("isbn")
    val isbn: String,
    @SerializedName("title")
    val title: String,
    @SerializedName("author")
    val author: String,
    @SerializedName("publisher")
    val publisher: String,
    @SerializedName("bookCoverImageUrl")
    val bookCoverImageUrl: String
)
